src/View/Amusements/index.html.twig

```diff
@@ -0,0 +1,31 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}
+<link rel="stylesheet" href="assets/css/amusements.css"> {% endblock %}
+
+{% block title %}{{ parent() }} - Attractions{% endblock %}
+
+
+{% block content %}
+
+<main class="container-fluid amusement-home pt-5">
+
+    
+    <h1 class="bg-primary section-title mx-auto mb-5 px-3 rounded-4">Attractions</h1>
+    {% for amusement in amusements %}
+        <div class="container rounded-4 bg-secondary my-4">
+            <div class="row amusement-card  ">
+                <div class="col-lg-9 d-flex flex-column justify-content-between {% if loop.index % 2 == 0 %} order-lg-2 {% endif %}">
+                    <h2 class="text-center mt-3"> {{amusement.name}} </h2>
+                    <p class="mb-4 text-md-center {% if loop.index % 2 == 0 %} text-lg-end {% else %} text-lg-start{% endif %} "> {{amusement.description}}
+                    </p>
+
+                </div>
+                <div class="col-lg-3 {% if loop.index % 2 == 0 %} order-lg-1 {% endif %}">
```

Comment:
tu peux faire un if loop.index is odd  ou is even (d'ailleurs loop.index commence à 1 et loop.index0 commence à zéro